library/src/main/kotlin/it/krzeminski/githubactions/dsl/Concurrency.kt

```diff
@@ -0,0 +1,16 @@
+package it.krzeminski.githubactions.dsl
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Using concurrency allow to run a workflow/job at a time
```

Comment:
Minor grammar issues:
```suggestion
 * Using concurrency allows running single workflow/job at a time
```
does it still preserve the meaning it should, after my fixes? :)

Reply:
yes it does :)